drivers/gles3/shader_gles3.cpp

```diff
@@ -486,6 +486,15 @@ RS::ShaderNativeSourceCode ShaderGLES3::version_get_native_source_code(RID p_ver
 	return source_code;
 }
 
+void ShaderGLES3::version_load(RID p_version) {
+	Version *version = version_owner.get_or_null(p_version);
+	ERR_FAIL_COND(!version);
+	if (!version->variants.is_empty()) {
+		return; // already loaded
```

Comment:
```suggestion
		return; // Already loaded.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).